verticaltabs.js

```diff
@@ -843,6 +843,25 @@ VerticalTabs.prototype = {
     }
   },
 
+  delayResizeTabs: function (delay) {
+    if (delay <= 0) {
```

Comment:
This was meant to normalize non-numeric values that get coerced to `0`. However, that doesn't actually matter much because `setTimeout` will do it internally anyway.

Really I would've wanted to use a default parameter, but wasn't sure if that's ok here, given that they're not used anywhere else in the addon, AFAICT.